packages/table/src/cell/editableCell.tsx

```diff
@@ -77,31 +82,46 @@ export class EditableCell extends React.Component<IEditableCellProps, IEditableC
     }
 
     public render() {
-        const { onCancel, onChange, onConfirm, ...spreadableProps } = this.props;
+        const { onCancel, onChange, onConfirm, truncated, wrapText, ...spreadableProps } = this.props;
 
         const { isEditing, dirtyValue, savedValue } = this.state;
         const interactive = spreadableProps.interactive || isEditing;
 
+        let cellContents: JSX.Element = null;
+        if (isEditing) {
+            cellContents = (
+                <EditableText
+                    isEditing={true}
+                    className={Classes.TABLE_EDITABLE_NAME}
+                    intent={spreadableProps.intent}
+                    minWidth={null}
+                    onCancel={this.handleCancel}
+                    onChange={this.handleChange}
+                    onConfirm={this.handleConfirm}
+                    onEdit={this.handleEdit}
+                    placeholder=""
+                    selectAllOnFocus={true}
+                    value={dirtyValue}
+                />
+            );
+        } else {
+            const textClasses = classNames({
+                [Classes.TABLE_TRUNCATED_TEXT]: truncated,
+                [Classes.TABLE_NO_WRAP_TEXT]: !wrapText,
+            });
+
+            cellContents = <div className={textClasses}>{savedValue}</div>;
+        }
+
         return (
             <Cell {...spreadableProps} truncated={false} interactive={interactive}>
                 <Draggable
                     onActivate={this.handleCellActivate}
                     onDoubleClick={this.handleCellDoubleClick}
-                    preventDefault={!interactive}
+                    preventDefault={false}
```

Comment:
Would appreciate a short comment as to why this is necessary.

Reply:
...because for some reason, on draggable, the default for preventDefault is _true_ . Which seems really dumb to me. If you know why this is, let me know?

Reply:
@cmslewis any more thoughts?

Reply:
Nah.

Reply:
@gscshoyru I meant a short comment in the code. :) No worries though.